src/mockhsm/commands.rs

```diff
@@ -305,6 +307,23 @@ fn get_object_info(state: &State, cmd_data: &[u8]) -> ResponseMessage {
     }
 }
 
+/// Get bytes of random data
+fn get_pseudo_random(_state: &State, cmd_data: &[u8]) -> ResponseMessage {
+    use rand::prelude::*;
+    use std::iter;
+
+    let command: GetPseudoRandomCommand = deserialize(cmd_data)
+        .unwrap_or_else(|e| panic!("error parsing CommandType::GetPseudoRandom: {:?}", e));
+
+    let mut rng = thread_rng();
+    let mut bytes = iter::repeat(0)
+        .take(command.bytes as usize)
+        .collect::<Vec<_>>();
```

Comment:
Simpler way to do the above is `vec![0u8; command.bytes as usize]`

Reply:
:+1: